tests/components/ozw/test_light.py

```diff
@@ -149,3 +149,335 @@ async def test_light(hass, light_data, light_msg, sent_messages):
     state = hass.states.get("light.led_bulb_6_multi_colour_level")
     assert state is not None
     assert state.state == "off"
+
+    # Test setting color_name
+    new_color = "blue"
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "color_name": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 9
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)